tutoraspects/templates/aspects/build/aspects-superset/openedx-assets/assets/datasets/ab_user.yaml

```diff
@@ -1,182 +1,200 @@
-  expression: ''
+  verbose_name: null
+- advanced_data_type: null
+  column_name: email
   description: null
-  python_date_format: null
+  expression: ''
   extra: null
-- column_name: email
-  verbose_name: null
-  is_dttm: false
+  filterable: true
+  groupby: true
   is_active: true
+  is_dttm: false
+  python_date_format: null
   type: VARCHAR(64)
-  groupby: true
-  filterable: true
-  expression: ''
+  verbose_name: null
+- advanced_data_type: null
+  column_name: active
   description: null
-  python_date_format: null
+  expression: ''
   extra: null
-- column_name: active
-  verbose_name: null
-  is_dttm: false
+  filterable: true
+  groupby: true
   is_active: true
+  is_dttm: false
+  python_date_format: null
   type: BOOLEAN
-  groupby: true
-  filterable: true
-  expression: ''
+  verbose_name: null
+- advanced_data_type: null
+  column_name: created_by_fk
   description: null
-  python_date_format: null
+  expression: ''
   extra: null
-- column_name: created_by_fk
-  verbose_name: null
-  is_dttm: false
+  filterable: true
+  groupby: true
   is_active: true
+  is_dttm: false
+  python_date_format: null
   type: INTEGER
-  groupby: true
-  filterable: true
-  expression: ''
+  verbose_name: null
+- advanced_data_type: null
+  column_name: changed_by_fk
   description: null
-  python_date_format: null
+  expression: ''
   extra: null
-- column_name: changed_by_fk
-  verbose_name: null
-  is_dttm: false
+  filterable: true
+  groupby: true
   is_active: true
+  is_dttm: false
+  python_date_format: null
   type: INTEGER
-  groupby: true
-  filterable: true
-  expression: ''
+  verbose_name: null
+- advanced_data_type: null
+  column_name: fail_login_count
   description: null
-  python_date_format: null
+  expression: ''
   extra: null
-- column_name: fail_login_count
-  verbose_name: null
-  is_dttm: false
+  filterable: true
+  groupby: true
   is_active: true
+  is_dttm: false
+  python_date_format: null
   type: INTEGER
-  groupby: true
-  filterable: true
-  expression: ''
+  verbose_name: null
+- advanced_data_type: null
+  column_name: login_count
   description: null
-  python_date_format: null
+  expression: ''
   extra: null
-- column_name: login_count
-  verbose_name: null
-  is_dttm: false
+  filterable: true
+  groupby: true
   is_active: true
+  is_dttm: false
+  python_date_format: null
   type: INTEGER
-  groupby: true
-  filterable: true
-  expression: ''
+  verbose_name: null
+- advanced_data_type: null
+  column_name: id
   description: null
-  python_date_format: null
+  expression: ''
   extra: null
-- column_name: id
-  verbose_name: null
-  is_dttm: false
+  filterable: true
+  groupby: true
   is_active: true
+  is_dttm: false
+  python_date_format: null
   type: INTEGER
-  groupby: true
-  filterable: true
-  expression: ''
+  verbose_name: null
+database_uuid: 06cd566f-e22a-4a79-b0e0-4c947de93ee0
+default_endpoint: null
+description: null
+extra:
+  certification:
+    certified_by: Data Analytics Team
```

Comment:
We should get rid of any of these we see, there are a few. Maybe the script can remove `certification` and `query_context` automatically?